gn/build/config/BUILDCONFIG.gn

```diff
@@ -70,11 +70,18 @@ if (_chip_defaults.custom_toolchain != "") {
     assert(false, "Unsupported target_cpu: ${target_cpu}")
   }
 } else if (target_os == "android") {
-  # TODO: Add x86, arm64
   if (current_cpu == "arm") {
+    android_abi = "armeabi-v7a"
```

Comment:
Prefer to move this variable to an import file, e.g. gn/build/android.gni

Variables here are global to all build files even with importing.

Reply:
Created android_abi.gni.